pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
 
-test_aspects:
+input_aspects:
   -  # (e.g. HTTP Messages, DOM Tree, CSS Styling, Accessibility Tree, Language, etc.,)
 
 authors:
   -  # (full names as found in `contributors` property in  `package.json` - if not yet listed, please have authors added to the list)
 ---
 
-## Test Procedure
-
-### Applicability
+## Applicability
 
 The rule applies to any (??) element ...
```

Comment:
Can we make it "This rule applies to..." since this is the wording we mostly use.
And should it be a comment instead, since this is not always the wording used?

Reply:
I would still like to see this changed, if I am to approve this PR.

Reply:
`The` is a definite article in English grammar used when the author/ reader knows exactly what is being referred to.

Eg: 
`It is the newest building in the town.`
`Who is the president of Madagascar?`

Given we are in the context of the rule. I believe `the` is the right usage over `this`.